docs/specification/attribute-templates.md

```diff
@@ -10,6 +10,9 @@ with an Attribute Template must conform to the bounds specified by the template.
 If the value is not compatible with the template (e.g. if the template specifies that
 the value must be in the range [0, 1] but the value is 2), then the data are invalid.
 
+![attribute templates](img/attribute-templates.png)
```

Comment:
This image as the "Confidential" footer in it.  I assume it is not confidential, and would suggest cropping it.